### DISCUSSION TITLE
Please add the VPS Latency to 'Network overview' like the WANs

### DISCUSSION THREAD

**Discussion by Fact0ryy** [Ysurac/openmptcprouter#3003] [Ideas]
Please add the VPS Latency to 'Network overview' like the WANs. Thx :-)

![image](https://github.com/Ysurac/openmptcprouter/assets/60034170/9718c35e-f2b6-4c92-a636-ba713ee3570c)

**Comment by Ysurac** [comment]
The problem is that latency to what ? From the VPS to internet, latency is not the same for each website/host.

**Comment by Fact0ryy** [comment]
Hi, thanks for the quick reply. I was thinking more about the real latency from the VPN to the router. So within the VPN connection. In the best case it will be the lowest latency from the fastest WAN, right?
